sdk/storage/Azure.Storage.Queues/src/QueueClient.cs

```diff
@@ -903,6 +904,72 @@ private void SetNameFieldsIfNull()
         #endregion ClearMessages
 
         #region EnqueueMessage
+        /// <summary>
+        /// Adds a new message to the back of a queue. The visibility timeout specifies how long the message should be invisible
+        /// to Dequeue and Peek operations. The message content must be a UTF-8 encoded string that is up to 64KB in size.
+        /// For more information, see <see href="https://docs.microsoft.com/en-us/rest/api/storageservices/put-message"/>.
+        /// </summary>
+        /// <param name="messageBytes">
+        /// The message bytes.
+        /// </param>
+        /// <param name="visibilityTimeout">
+        /// Visibility timeout.  Optional with a default value of 0.  Cannot be larger than 7 days.
+        /// </param>
+        /// <param name="timeToLive">
+        /// Optional. Specifies the time-to-live interval for the message
+        /// </param>
+        /// <param name="cancellationToken">
+        /// Optional <see cref="CancellationToken"/>.
+        /// </param>
+        /// <returns>
+        /// <see cref="Response{EnqueuedMessage}"/>
+        /// </returns>
+        public virtual Response<EnqueuedMessage> EnqueueMessage(
+            byte[] messageBytes,
+            TimeSpan? visibilityTimeout = default,
+            TimeSpan? timeToLive = default,
+            CancellationToken cancellationToken = default) =>
+            EnqueueMessageInternal(
+                Convert.ToBase64String(messageBytes),
+                visibilityTimeout,
```

Comment:
I think this name is not super descriptive.

Reply:
`visibilityTimeout`?  That's a [service level concept](https://docs.microsoft.com/en-us/rest/api/storageservices/put-message#uri-parameters).  We could reconsider the name, but it's a big change across a lot of the Queue APIs and is outside the scope of this PR.  Please file an issue if you want to do a global rename here.

Reply:
If it's a big change, then it's not worth it.